src/components/2019/year-in-review/News.js

```diff
@@ -0,0 +1,534 @@
+                                    <Date inverse={true}> Nov. 18 </Date>
+                            </div>
+                            <div class="col-md-6">
+                                <img class="img-fluid"
+                                    src="https://snworksceo.imgix.net/dpn/0d4576c9-3211-4213-b1c9-20068b954eeb.sized-1000x1000.jpg?w=1000" />
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <MoreStories inverse={true}>
+                    <SubSectionHeader color={"#131313"}>MORE STORIES</SubSectionHeader>
+                    <div class="row">
+                        <div class="col-md-3 right-border-inverse">
+                            <StyledStoryLink href="https://www.thedp.com/article/2019/02/penn-design-stuart-weitzman-donation-fashion-philadelphia"  >
+                                <SecondaryStory inverse={true}> PennDesign renamed after fashion icon Stuart Weitzman — relieving financial uncertainty </SecondaryStory>
+                            </StyledStoryLink>
+                            <Date inverse={true}> Feb. 26 </Date>
+                        </div>
+                        <div class="col-md-3 right-border-inverse narrow-col-left">
+                            <StyledStoryLink href="https://www.thedp.com/article/2019/11/penn-law-carey-foundation-donation-renaming"  >
+                                <SecondaryStory inverse={true}> Penn Law renamed ‘Carey Law School’ following record $125 million donation </SecondaryStory>
+                            </StyledStoryLink>
+                            <Date inverse={true}> Nov. 8 </Date>
+                        </div>
+                        <div class="col-md-3 right-border-inverse narrow-col-left">
+                            <StyledStoryLink href="https://www.thedp.com/article/2019/11/penn-carey-law-ruger-rename-backlash-student-alumni"  >
+                                <SecondaryStory inverse={true}> 'A complete nightmare': Penn Law students and alums reflect on name-change process </SecondaryStory>
+                            </StyledStoryLink>
+                            <Date inverse={true}> Nov. 19 </Date>
+                        </div>
+                        <div class="col-md-3 narrow-col-left">
+                            <StyledStoryLink href="https://www.thedp.com/article/2019/11/penn-law-carey-ruger-rename-town-hall"  >
+                                <SecondaryStory inverse={true}> Admin considers changing 'Carey Law' back to 'Penn Law' after student backlash </SecondaryStory>
+                            </StyledStoryLink>
+                            <Date inverse={true}> Nov. 12 </Date>
+                        </div>
+                    </div>
+                </MoreStories>
+            </div>
+        </Section>
+
+        {/* NEWS 3 */}
```

Comment:
either add another comment for the sections above or remove this? Or abstract out each section into a seperate functional component.